packages/shared/src/components/profile/LanguageDropdown.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { ReactElement, useState } from 'react';
+import classNames from 'classnames';
+import { Dropdown, DropdownClassName } from '../fields/Dropdown';
+import { LanguageIcon } from '../icons';
+import { BaseFieldProps } from '../fields/BaseFieldContainer';
+import { ContentLanguage, contnetLanguageToLabelMap } from '../../lib/user';
+import { withExperiment } from '../withExperiment';
+import { feature } from '../../lib/featureManagement';
+import { IconProps } from '../Icon';
+
+type ClassName = {
+  hint?: string;
+  dropdown?: string;
+} & DropdownClassName;
+
+type Props = {
+  className?: ClassName;
+  defaultValue?: ContentLanguage;
+  onChange?: (value: ContentLanguage, index: number) => void;
+  icon?: ReactElement<IconProps>;
+} & Pick<BaseFieldProps, 'name' | 'valid' | 'hint' | 'saveHintSpace'>;
+
+const defaultIcon = <LanguageIcon className="ml-0 mr-1" />;
+
+const LanguageDropdownDefault = ({
+  className = {},
+  onChange,
+  hint = '',
+  valid = true,
+  defaultValue,
+  name,
+  saveHintSpace = false,
+  icon = defaultIcon,
+}: Props): ReactElement => {
+  const [open, setOpen] = useState(false);
+  const [selectedIndex, setSelectedIndex] = useState(
+    defaultValue ? Object.values(ContentLanguage).indexOf(defaultValue) : 0,
+  );
+
+  const {
+    hint: hintClassName,
+    label: labelClassName,
+    button: buttonClassName,
+    menu: menuClassName,
+    item: itemClassName,
+    container: containerClassName,
+    dropdown: dropdownClassName,
+  } = className;
+
+  return (
+    <div
+      className={classNames(
+        'relative flex flex-col items-stretch',
+        containerClassName,
+      )}
+    >
+      <Dropdown
+        className={{
+          label: classNames(labelClassName, 'typo-body'),
+          button: classNames(
+            buttonClassName,
+            '!px-3',
+            !open &&
+              !valid &&
+              '!shadow-[inset_0.125rem_0_0_var(--status-error)]',
```

Comment:
Lol we need a better way to do this.
But basically it just copies from exp level dropdown right?

Can we log a backlog item to consolidate the styling otherwise? *(not important/urgent)

Reply:
AS-533